src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/MongoDb/MongoDbInstrumentMethodAttribute.cs

```diff
@@ -0,0 +1,15 @@
+using Datadog.Trace.ClrProfiler.Integrations;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.MongoDb
+{
+    internal abstract class MongoDbInstrumentMethodAttribute : InstrumentMethodAttribute
+    {
+        protected MongoDbInstrumentMethodAttribute(string typeName)
+        {
+            AssemblyName = MongoDbIntegration.MongoDbClientAssembly;
+            TypeName = typeName;
+            IntegrationName = MongoDbIntegration.IntegrationName;
+            ParameterTypeNames = new[] { "MongoDB.Driver.Core.Connections.IConnection", ClrNames.CancellationToken };
```

Comment:
nit: Would you mind moving the `ParameterTypeNames` assignment into each of the method-specific attributes? That could reduce confusion if we later decided to instrument other methods in MongoDb

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1214/commits/ff06266d093c73f6d518c8aca8c7d2017e203936